src/appmixer/monday/core/AddFileToUpdate/AddFileToUpdate.js

```diff
@@ -0,0 +1,29 @@
+const FormData = require('form-data');
+
+/**
+ * Component for making API requests.
+ * @extends {Component}
+ */
+module.exports = {
+
+    async receive(context) {
+
+        const { fileId, fileName, updateId } = context.messages.file.content;
+        const query = 'mutation add_file_to_update($file: File!) { add_file_to_update (update_id: ' + updateId + ', file: $file) {id}}';
```

Comment:
Sorry for not highlighting in the previous review, but this query should probably also go to the queries.js file

Reply:
I could have notice this... 
done